api_docs/README.md

```diff
@@ -4,6 +4,6 @@ To generate new docs:
 
 ```
 $ cd api_docs
-$ npm install
-$ npx dociql config.yml
+$ docker build -t dociql .
+$ docker run -v $PWD/public:/src/public --net=host -it --rm dociql npx dociql config.yml
```

Comment:
Teniendo ese `CMD` en el `Dockerfile`, ¿no debería ser suficiente con...

```suggestion
$ docker run -v $PWD/public:/src/public --net=host -it --rm dociql
```

?

Reply:
AFAIK, the CMD is overrun for whatever argument you pass to `docker run`. So, I think @RodrigoMolinaMadrid approach is correct.

Reply:
I know, that's why I see one of them unnecessary. But if you think it's ok to have both, it's ok for me.